cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -2586,8 +2628,23 @@ public TopicsChangedListener getTopicsChangedListener() {
 
     @Override
     public void onRequestPermissionsResult(int requestCode, @NonNull String[] permissions, @NonNull int[] grantResults) {
-        if (requestCode == locationPermissionRequestCode && (geofenceList == null || geofenceList.size() == 0)) {
-            this.initGeoFences();
+        switch (requestCode) {
+            case (pushNotificationPermissionRequestCode):
+                if (this.hasNotificationPermission() && !this.ignoreDisabledNotificationPermission) {
+                    this.subscribe(true, pendingSubscribeCallbackListener);
+                } else if (!this.ignoreDisabledNotificationPermission) {
+                    String error = "Can not subscribe because the notification permission has been denied by the user. You can call CleverPush.setIgnoreDisabledNotificationPermission(true) to still allow subscriptions, e.g. for silent pushes.";
```

Comment:
do we have a spot for texts like these in android sdk? if not atleast create a const at the beginning of the file for it